desktop/dashboard.md

```diff
@@ -227,23 +227,40 @@ To interact with remote repositories:
 
 ## Explore volumes
 
-The **Volumes** view in Docker Dashboard enables you to easily create and delete [volumes](../storage/volumes.md) and see which ones are being used. If you are a developer subscribed to a Pro or a Team plan, you can also see which container is using a specific volume and explore the files and folders in your volumes.
+A [volume](../storage/volumes.md) is a directory on a disk or in a container. Volumes are directly managed by Docker and are stored as part of the host file system.
```

Comment:
It’s actually the other way around: volumes are directories stored in the VM (also but rarely called the guest file system, mostly internally at Docker I think) while bind mounts are directories (or files) mounted from the host.

Reply:
Updated the description. PTAL and let me know if we should further amend this.

Reply:
Yes, that looks fine. The only thing I’m wondering is if users will know what’s the VM here, do the docs talk a lot about it in general? Do we not try to hide these kinds of implementation details?

Reply:
Was trying to provide some sort of intro here with a link to the Volumes section for details.

Reply:
Maybe something like «Volumes can be used to store files and share them among containers.»?

Reply:
Or perhaps also mentioning volumes outlive containers?